pydantic/networks.py

```diff
@@ -114,7 +115,7 @@ class AnyUrl(str):
 
     @no_type_check
     def __new__(cls, url: Optional[str], **kwargs) -> object:
-        return str.__new__(cls, cls.build(**kwargs) if url is None else url)
+        return str.__new__(cls, cls.build(**kwargs) if url is None else url)  # noqa
```

Comment:
I would remove the `# noqa`. I understand some IDEs like Pycharm will display some kind of warning but as long as `flake8` says nothing we're good